lmn/templates/lmn/shows/show_detail.html

```diff
@@ -1,8 +1,22 @@
 {% extends 'lmn/base.html' %}
 {% block content %}
 
-<h2>Show Detail</h2>
+<h2>{{ show.artist.name }} @ {{ show.venue.name }} {{ show.show_date }}</h2>
+<h2>Notes:</h2>
+{% for note in notes %}
 
+  <div class="show" id="show_{{ show.pk }}">
+    <p>
+      <a href="{% url 'note_detail' note_pk=note.pk %}">{{ note.title }}</a>
+    </p>
+  </div>
+
+{% empty %}
+
+<p>No Notes Yet</p>
+
+{% endfor %}
+<a href="{% url 'new_note' show_pk=show.pk %}">Add a Note:</a>
```

Comment:
Add a note shouldn't have a colon at the end, but not still functional, so I'm calling it good.